### PR TITLE
Initial Mulesoft plugin release

### PR DESCRIPTION
**Pull request by Castyr**
Ultimate Software is requesting to add our open source MuleSoft coverage plugin to the marketplace. You can find the request with more links in the Community forms here https://community.sonarsource.com/t/new-release-mulesoft-coverage-0-5-1/14890

### CODE REVIEW

File: mulesoft.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+privateVersions=
+
+defaults.mavenGroupId=com.ultimatesoftware.sonar.plugins.mulesoft
+defaults.mavenArtifactId=mulesoftplugin
+
+0.5.1.description=Initial release, includes basic support for analyzing Mulesoft test coverage reports.
+0.5.1.sqlVersions=[7.3,*]
+0.5.1.downloadUrl=https://search.maven.org/remotecontent?filepath=com/ultimatesoftware/sonar/plugins/mulesoft/mulesoftplugin/0.5.1-RELEASE/mulesoftplugin-0.5.1-RELEASE.jar
+0.5.1.date=2019-09-30
+0.5.1.changelogUrl=https://github.com/UltimateSoftware/sonar-mulesoft#release-051
```
Comment (ganncamp) [inline mulesoft.properties]: GH seems to want a newline at the end here.

File: mulesoft.properties [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+name=Mulesoft
+publicVersions=0.5.1
+archivedVersions=
+privateVersions=
+
+defaults.mavenGroupId=com.ultimatesoftware.sonar.plugins.mulesoft
+defaults.mavenArtifactId=mulesoftplugin
+
+0.5.1.description=Initial release, includes basic support for analyzing Mulesoft test coverage reports.
+0.5.1.sqlVersions=[7.3,*]
```
Comment (ganncamp) [inline mulesoft.properties]: `*` is not valid as the end of the compatibility range. Use `LATEST` instead

File: mulesoft.properties (hunk 1 above)
Comment (Castyr) [reply mulesoft.properties]: Added the new line. 🤦

File: mulesoft.properties (hunk 2 above)
Comment (Castyr) [reply mulesoft.properties]: This is fixed.

**Comment by Castyr** [comment]
I have made the requested changes.

**Pull request merged**